dbscripts/xml/upgrade/3.0.0_postupgrade_galley_files.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * 3.0.0_postupgrade_galley_files.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * 3.0.0.0 updates for galley files.
+  -->
+<data>
+	<sql>
+		<query>UPDATE submission_files sf, submission_galleys sg SET sf.assoc_type = 521, sf.assoc_id = sg.galley_id WHERE sf.file_id = sg.file_id AND sf.submission_id = sg.submission_id</query>
```

Comment:
When you code magic numbers in for SQL queries, make sure you add a comment to the end of the line:

    <!-- ASSOC_TYPE_... -->

(where `...` is the appropriate constant name.)

That way it's still possible to grep for all occurrences of a constant, if it should change in the future or something.